terraform/init.tf

```diff
@@ -8,6 +8,15 @@ terraform {
   }
 }
 
-variable "cloudfront_zone_id" {
-  default = "Z33AYJ8TM3BH4J"
+locals {
+  // Unclear where this value comes from; presumably it's a deprecated value from AWS. -Aidan Feldman, 11/6/17
+  old_cloudfront_zone_id = "Z33AYJ8TM3BH4J"
+
+  // https://docs.aws.amazon.com/general/latest/gr/rande.html#cf_region
+  cloudfront_zone_id = "Z2FDTNDATAQYW2"
+  // https://docs.aws.amazon.com/general/latest/gr/rande.html#elb_region
+  elb_zone_id = "Z35SXDOTRQ7X7K"
+
+  dmarc_10 = "v=DMARC1; p=none; pct=10; fo=1; ri=86400; rua=mailto:dmarcreports@gsa.gov,mailto:reports@dmarc.cyber.dhs.gov; ruf=mailto:dmarcfailures@gsa.gov"
+  dmarc_100 = "v=DMARC1; p=none; pct=100; fo=1; ri=86400; rua=mailto:dmarcreports@gsa.gov,mailto:reports@dmarc.cyber.dhs.gov; ruf=mailto:dmarcfailures@gsa.gov"
```

Comment:
Not sure if it's intentional that we use a mix of these two, but we can clear that up later if the answer is "no". /cc @konklone @boberlas

Reply:
There _shouldn't_ be a need to mix `pct=10` and `pct=100` for `p=none` headers across our sites. And we already hit the deadline for having `pct=100` in place for `p=none` across GSA.

There may well be a need to mix `pct=10` and `pct=100` for `p=reject` through some of 2018, though -- that's not what these policies are, and I would ping a service using `pct=10` before upping it to 100 on them in any case, but flagging that gradual DMARC adoption is appropriate for some other policy variants we may want to deploy across our DNS this year.

Reply:
Cool, will do as a follow-up. #213